contracts/zap/staking/SolidZapStaker.sol

```diff
@@ -20,14 +20,14 @@ contract SolidZapStaker is ISolidZapStaker, ReentrancyGuard {
     address public immutable iUniProxy;
     address public immutable solidStaking;
 
-    struct HypervisorTokens {
-        address token0;
-        address token1;
+    struct SwapResult {
+        address _address;
```

Comment:
Is there a reason for `_address` instead of `address`?

Reply:
yeah, because `address` is a reserved keyword

Reply:
yeah of course